client/js/components/input.js

```diff
@@ -9,6 +9,7 @@ const Input = ({ classes, input, type, placeholder }) => {
 			<input
 				className={classes.textInput}
 				type={type}
+				style={{ fontSize: input.name === 'prcName' ? '30px' : '18px' }}
```

Comment:
This can actually be done in JSS. If you create a new property on the styles object with a text size of `30px` then you can apply the styles accordingly.